src/js/components/Grommet/Grommet.js

```diff
@@ -134,11 +140,12 @@ const Grommet = forwardRef((props, ref) => {
     theme.global.deviceBreakpoints.tablet;
 
   const grommetRef = useForwardedRef(ref);
+  const getRootsContext = useCallback((root) => [root], []);
```

Comment:
This one seems a bit odd as there would ideally be a way to have something that doesn't change across renders if grommetRef.current doesn't change, but we're going to call this function on every render here. So, we are going to be creating a new Array on each render. I think something that hits the spirit of this more would be to `const [rootsContextValue, setRootsContextValue] = useState()` and then call `setRootsContextValue` within the StyledGrommet `ref`, something like `ref={(node) => setRootsContextValue([node])}`.

Reply:
Addressed.